packages/flutter/test/material/dropdown_test.dart

```diff
@@ -3678,4 +3678,56 @@ void main() {
 
     expect(tester.takeException(), null);
   });
+
+  group('BorderRadius property works properly for DropdownButtonFormField', () {
+    const double radius = 20;
+
+    Widget build({List<String>? items, String? value, BorderRadius? borderRadius}) {
+      return MaterialApp(
+        home: Scaffold(
+          body: Center(
+            child: DropdownButtonFormField<String>(
+              borderRadius: borderRadius,
+              value: value,
+              items: items!
+                .map<DropdownMenuItem<String>>((String value) {
+                  return DropdownMenuItem<String>(
+                    value: value,
+                    child: Text(value),
+                  );
+              }).toList(),
+              onChanged: (_) { },
+            ),
+          ),
+        ),
+      );
+    }
+
+    testWidgets('When multiple items are available', (WidgetTester tester) async {
```

Comment:
Try to test this the same way `DropdownButton.borderRadius` is tested.

<details>
<summary>reference test</summary>

https://github.com/flutter/flutter/blob/39cdeb59621b144acd7424607c3bb80af20eb4a1/packages/flutter/test/material/dropdown_test.dart#L3528-L3573
</details>